pom.xml

```diff
@@ -31,136 +31,4 @@ limitations under the License.
 
```

Comment:
pom.xml should be removed from the project, it should be contained only in the generator project